package.json

```diff
@@ -22,7 +23,7 @@
         "test-integ": "npm run compile && npm run test-integ --workspaces --if-present",
         "test-unit": "npm run compile && npm run test-unit --workspaces --if-present",
         "update-runtimes": "cp ../language-server-runtimes/aws-*.tgz bin/ && npm update @aws/language-server-runtimes",
-        "package": "npm install && npm run compile && npm run package-x64 --workspaces --if-present"
+        "package": "npm install && npm run compile && npm run package-x64 --workspaces --if-present && npm run package --workspaces --if-present"
```

Comment:
nit: we should refactor all the packages to just use `npm run package`, just makes the script too long here

Reply:
Maybe we should: I did not see any usage of `package` & `package-x64` in one package. But it's a very low priority. But it' a low priority.